tests/polarion/test_polarion_set_automated_coverage.py

```
@@ -0,0 +1,435 @@
+                        "--previous-commit",
+                        "abc123",
+                        "--current-commit",
+                        "def456",
+                        # No --project-id provided
+                    ],
+                )
+
+                # Assert
+                assert result.exit_code == 0
+                # Check that get_polarion_project_id was called with default config path
+                args, kwargs = mock_get_project.call_args
```

Ruff (0.14.4)

Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)